### ISSUE TITLE
Dashboard: English text on French page

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#98]
URL: https://hcdev.openplus.ca/fr/user/197
User Agent: Chrome
Language: French

Issue Description: There is English text on the French page. Please see screenshot.  All text should contain the language of the page.

Steps to reproduce: Visually scan the Dashboard page

Expected results: All text should contain the correct language of the page.
![image](https://user-images.githubusercontent.com/68117281/136403183-5ab2678f-eefb-40b0-bdd6-e1005183c2f6.png)

**Comment by echouin** [comment]
@mmiron-HC this is a duplicate entry. I have already tasked the fix for Issue #30 (same item): https://github.com/mmiron-HC/WST-RSAMS/issues/30#issue-960545855

**Comment by echouin** [comment]
As noted, Issue #30 has now been resolved. 30 and 98 (duplicate) should be closed at the same time. Thank you.

**Comment by ndolter** [comment]
Closing ticket since this is a duplicate.
